ee/webhooks/cmd/fx-modules/fx-modules.go

```diff
@@ -0,0 +1,25 @@
+package fxmodules
+
+import (
+	"fmt"
+	"net/http"
+
+	"go.opentelemetry.io/contrib/instrumentation/net/http/otelhttp"
+
+	"go.opentelemetry.io/otel"
+)
+
+var Tracer = otel.Tracer("webhook")
+
+func FxProvideHttpClient() *http.Client {
+	return &http.Client{
+		Transport: otelhttp.NewTransport(http.DefaultTransport, otelhttp.WithSpanNameFormatter(func(operation string, r *http.Request) string {
+			str := fmt.Sprintf("%s %s", r.Method, r.URL.Path)
```

Comment:
Avoid this if you have some kind of id in paths, like uuid. (I have not checked)
Span names are used as bucket.
If it changes always, the tooling cannot do the right job.
Sometimes, for some tools, it has a big performance impact.

Reply:
resolve